adafruit_as726x.py

```diff
@@ -527,20 +536,20 @@ class AS726x_UART(AS726x):
 
     """
 
-    def __init__(self, uart):
+    def __init__(self, uart: busio.UART):
         self._uart = uart
         self._uart.baudrate = 115200
         super().__init__()
 
-    def read_channel(self, channel):
+    def read_channel(self, channel: int) -> int:
```

Comment:
This isn't a comment but a question - though this uses `_virtual_read()` it's return type is `int` because it isn't one of the calibrated channels that casts the result as a `float`, am I understanding that correctly?  So it should be reasonably expected to be `int` even though the branching of `_virtual_read()` has a `float` return.

Reply:
Typically anything that returns `float` can be expected to return `int` as well, so `float` should be good!

Reply:
It's basically it's on `Union`, is my understanding